types/src/endpoints.ts

```diff
@@ -251,980 +226,994 @@ export type BatchEditApplicationCommandPermissionsEndpoint<
 > = `/applications/${ApplicationID}/guilds/${GuildID}/commands/permissions`;
 
 /**
- * Get Entitlements
- * @method GET
+ * Create Webhook
+ * @method POST
  */
-export type GetEntitlementsEndpoint<
-  ApplicationID extends snowflake = snowflake,
-> = `/applications/${ApplicationID}/entitlements`;
+export type CreateWebhookEndpoint<
+  ChannelID extends snowflake = snowflake,
+> = `/channels/${ChannelID}/webhooks`;
 
 /**
- * Get Entitlement
+ * Get Channel Webhooks
  * @method GET
  */
-export type GetEntitlementEndpoint<
-  ApplicationID extends snowflake = snowflake,
-  EntitlementID extends snowflake = snowflake,
-> = `/applications/${ApplicationID}/entitlements/${EntitlementID}`;
+export type GetChannelWebhooksEndpoint<
+  ChannelID extends snowflake = snowflake,
+> = `/channels/${ChannelID}/webhooks`;
 
 /**
- * Get SKUs
+ * Get Guild Webhooks
  * @method GET
  */
-export type GetSKUsEndpoint<
-  ApplicationID extends snowflake = snowflake,
-> = `/applications/${ApplicationID}/skus`;
-
-/**
- * Consume SKU
- * @method POST
- */
-export type ConsumeSKUEndpoint<
-  ApplicationID extends snowflake = snowflake,
-  EntitlementID extends snowflake = snowflake,
-> = `/applications/${ApplicationID}/entitlements/${EntitlementID}/consume`;
-
-/**
- * Delete Test Entitlement
- * @method DELETE
- */
-export type DeleteTestEntitlementEndpoint<
-  ApplicationID extends snowflake = snowflake,
-  EntitlementID extends snowflake = snowflake,
-> = `/applications/${ApplicationID}/entitlements/${EntitlementID}`;
+export type GetGuildWebhooksEndpoint<
+  GuildID extends snowflake = snowflake,
+> = `/guilds/${GuildID}/webhooks`;
 
 /**
- * Create Purchase Discount
- * @method PUT
+ * Get Webhook
+ * @method GET
  */
-export type CreatePurchaseDiscountEndpoint<
-  SkuID extends snowflake = snowflake,
-  UserID extends snowflake = snowflake,
-> = `/store/skus/${SkuID}/discounts/${UserID}`;
+export type GetWebhookEndpoint<
+  WebhookID extends snowflake = snowflake,
+> = `/webhooks/${WebhookID}`;
 
 /**
- * Delete Purchase Discount
- * @method DELETE
+ * Get Webhook with Token
+ * @method GET
  */
-export type DeletePurchaseDiscountEndpoint<
-  SkuID extends snowflake = snowflake,
-  UserID extends snowflake = snowflake,
-> = `/store/skus/${SkuID}/discounts/${UserID}`;
+export type GetWebhookwithTokenEndpoint<
+  WebhookID extends snowflake = snowflake,
+  WebhookToken extends string = string,
+> = `/webhooks/${WebhookID}/${WebhookToken}`;
 
 /**
- * Get Guild Audit Log
- * @method GET
+ * Edit Webhook
+ * @method PATCH
  */
-export type GetGuildAuditLogEndpoint<
-  GuildID extends snowflake = snowflake,
-> = `/guilds/${GuildID}/audit-logs`;
+export type EditWebhookEndpoint<
```

Comment:
Nit: This is effectively Duplicate code. Is there some way we can undupe this without making the naming worse (`GetOrEditWebhook`)

Reply:
this is auto-generated code so later it can be renamed